### PR TITLE
Memcached not working in workspace with php 7.

### PR DESCRIPTION
**Pull request by iamsaqibs**
Not Working Memcached in workspace with php 7

I have added bellow code to workspace dockerfile.
```
###########################################################################
# Memcached Dependecies:
###########################################################################

RUN if [ ${PHP_FPM_INSTALL_MEMCACHED} = true ]; then \
  apt-get -y install php${LARADOCK_PHP_VERSION}-igbinary \
  && apt-get -y install php${LARADOCK_PHP_VERSION}-memcached \
;fi
```

### CODE REVIEW

File: workspace/Dockerfile [hunk 1]
```diff
@@ -1220,6 +1220,16 @@ RUN if [ ${INSTALL_TERRAFORM} = true ]; then \
     && mv terraform /usr/local/bin \
     && rm terraform_0.10.6_linux_amd64.zip \
 ;fi
+
+###########################################################################
+# Memcached Dependecies:
+###########################################################################
+
+RUN if [ ${PHP_FPM_INSTALL_MEMCACHED} = true ]; then \
```
Comment (bestlong) [inline workspace/Dockerfile:1228]: ARG is missing and  this is workspace, not php-fpm.

Comment (iamsaqibs) [reply workspace/Dockerfile]: Oh my bad. I have added ARG for workspace.

**Comment by bestlong** [comment]
https://github.com/laradock/laradock/issues/2844#issuecomment-791967260

**Comment by bestlong** [comment]
In Dockerfile should be

```
ARG INSTALL_MEMCACHED=false

RUN if [ ${INSTALL_MEMCACHED} = true ]; then \
...
```

**Comment by iamsaqibs** [comment]
@bestlong I have modified Dockerfile as you said.

**Pull request merged**